CONTRIBUTING.rst

```diff
@@ -174,16 +174,17 @@ To build the documentation you'll need `Sphinx <http://www.sphinx-doc.org/>`_,
 To install (and activate) a `conda environment`_ named ``notebook_docs``
 containing all the necessary packages (except pandoc), use::
 
-    conda env create -f docs/environment.yml
+    conda create -n notebook_docs pip
     conda activate notebook_docs  # Linux and OS X
-    activate notebook_docs         # Windows
+    activate notebook_docs        # Windows
+    pip install .[docs]
 
 .. _conda environment:
     https://conda.io/docs/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file
```

Comment:
This could go away since we're no longer using it.

Reply:
:+1: